src/components/Album.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { Artist } from "./Artist"
+import { AlbumName } from "./AlbumName"
+import { Cover } from "./Cover";
+import "./styling/album.css"
+
+
+//get the data *mArray* and "filter" it further down (albums)
+
+export const Album = ({ mArray }) => {
+	let all = mArray.albums.items;
+	//set default state
+	const [selected, setSelected] = useState(all);
```

Comment:
it's good to see how to combine useState and array method

Reply:
yup  it was the easiest way to handle the interactivity with the buttons :)

Reply:
Agree! 👍